packages/theme-chalk/src/checkbox-button.scss

```
@@ -24,14 +24,16 @@
       #{getCssVarName('button-bg-color')},
       map.get($button, 'bg-color')
     );
-    outline: getCssVar('border');
+    border: getCssVar('border');
```

Stylelint (17.12.0)

[error] Expected "getCssVar" to be "getcssvar" (function-name-case)

(function-name-case)